src/components/starlight/MarkdownContent.astro

```diff
@@ -0,0 +1,20 @@
+---
+import type { Props } from '@astrojs/starlight/props';
+import '@astrojs/starlight/style/markdown.css';
+
+import { Aside } from '@astrojs/starlight/components';
+import { coerce, satisfies } from 'semver'
```

Comment:
nit: I see why you grouped the imports like this. I like to group them by type and others.

Reply:
Yeah, this was based on what was from the initial component, before overriding, but this can all be changed. I'm guessing if we end up adding a linter, that will sort this out for us

Reply:
Yeah I think it does not matter just yet